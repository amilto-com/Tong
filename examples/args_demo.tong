// Simple demonstration of the built-in 'args' module
let args = import("args")

print("script:", args.script)
print("argc:", args.len())
print("all:", args.all)

// Flags and key-values
print("has --verbose:", args.has("--verbose"))
print("mode:", args.value("--mode"))

// Positional argument access
print("first arg:", args.get(0))
