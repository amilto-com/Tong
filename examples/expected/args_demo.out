script: examples/args_demo.tong
argc: 0
all: [examples/args_demo.tong]
has --verbose: false
mode: 
first arg: 
